apps/settings/js/panels/about/update_check.js

```diff
@@ -150,7 +154,8 @@ define(function(require) {
      */
     _checkForUpdates: function uc__checkForUpdates() {
       if (!navigator.onLine) {
-        alert(this._('no-network-when-update'));
+        navigator.mozL10n.formatValue('no-network-when-update').then(
+          msg => alert(msg));
```

Comment:
Yep, it's a good idea to disable the button before formatValue resolves, I assume usually users won't notice this disabled state though. Would it be that much a 5s delay?